src/internal/api/components/list.go

```diff
@@ -0,0 +1,14 @@
+package components
+
+import (
+	"net/http"
+
+	"github.com/defenseunicorns/zarf/src/config"
+	"github.com/defenseunicorns/zarf/src/internal/api/common"
+)
+
+// ListDeployedPackages writes a list of packages that have been deployed to the connected cluster.
+func ListDeployedComponents(w http.ResponseWriter, r *http.Request) {
```

Comment:
Confused we're using "deployed" and "deploying" interchangeably here, think we need to pick one.

Reply:
Is it possibly deployed = complete state
Deploying = in progress state?

Reply:
If it were different functions that would be ok, but agree with @jeff-mccoy that in the same function it should be one or the other.